components/developer/jenkins/jenkins-core-weekly.p5m

```diff
@@ -0,0 +1,34 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source.  A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016-2017 Jim Klimov
+#
+
+set name=pkg.fmri value=pkg:/$(COMPONENT_FMRI)@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
+set name=pkg.summary value="$(COMPONENT_SUMMARY) - weekly core build"
+set name=info.classification value="$(COMPONENT_CLASSIFICATION)"
+set name=info.upstream-url value=$(COMPONENT_PROJECT_URL)
+set name=info.source-url value=$(COMPONENT_ARCHIVE_URL)
+set name=org.opensolaris.consolidation value=$(CONSOLIDATION)
+
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+# Note: java8+ is required since 2017
+# This may be JRE8 for the server, though JDK8 also suffices :)
+depend fmri=__TBD pkg.debug.depend.file=usr/bin/java type=require
+#depend fmri=__TBD pkg.debug.depend.file=usr/bin/javac type=require
+
+file path=usr/share/jenkins/war-weekly/$(COMPONENT_NAME)-$(COMPONENT_VERSION).war
+link path=usr/share/jenkins/war/jenkins.war \
+     target=../war-weekly/$(COMPONENT_NAME)-$(COMPONENT_VERSION).war \
+     mediator=jenkins-core-weekly \
```

Comment:
Does it make sense to mediate this? Normally, people will run either LTS or the latest version. Also what happens when you use plugins, but they are not compatible with the version configured in the mediator? Do we care about those problems? To me, it would make sense just to have updates shipped by "pkg update"